### ISSUE TITLE
PSM is Enabled by default which shutsdown the modem 

### ISSUE THREAD

**Issue by akankshaashwini** [legatoproject/legato-af#46]
With MCU version 002.011 PSM is enabled by default which shutsdown the modem after ~40 minutes:
root@swi-mdm9x28-wp:/data/psm# cat ./psm_nv_backup.txt
PSM_ENABLED=1
PSM_THRESHOLD=60
PSM_OOS_DURATION=86400
PSM_RANDOMIZATION_WINDOW=5
PSM_PERIODIC_TAU_TIMER=11520
PSM_PERIODIC_TAU_RCVD_TIME=2936
PSM_APP_ACTIVE_TIME=0
PSM_BOOTUP_DELAY=3 

Checked with a previous version 002.009 and found it to be disabled:
root@swi-mdm9x28-wp:~# cat /data/psm/psm_nv_backup.txt
PSM_ENABLED=0
PSM_THRESHOLD=0
PSM_OOS_DURATION=0
PSM_RANDOMIZATION_WINDOW=0
PSM_PERIODIC_TAU_TIMER=0
PSM_PERIODIC_TAU_RCVD_TIME=0
PSM_APP_ACTIVE_TIME=0
PSM_BOOTUP_DELAY=-1

**Comment by Redferne** [comment]
It is possible to disable PSM directly on the modem using:
```
root@swi-mdm9x28-wp:~# microcom /dev/ttyAT
at+cpsms=0
OK
```
Exit microcom by CTRL+a and then CTRL+x

**Comment by vondraussen** [comment]
Good point, we could probably reconfigure it via AT. However we need PSM, so deactivating it wouldn't help us much.
We're switching back to 19.02. which works for us.

It's still an issue that it is activated by default.
